.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
We are moving to run pre-commit in prow. For example https://github.com/openshift/release/pull/35600

Reply:
Thank you this one is helpful. :)